src/Forms/Tip.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use InvalidArgumentException;
+
+/**
+ * Represents a Tip which can be rendered alongside a form field in the front-end.
+ * See the Tip component in the silverstripe/admin module.
+ *
+ * @package SilverStripe\Forms
+ */
+class Tip
+{
+    /**
+     * These map to levels in the front-end Tip component
+     */
+    const IMPORTANCE_LEVELS = [
```

Comment:
I would have made them two individual constants, but this way works as well.

Reply:
This more closely aligns to the front-end implementation (I agree that JS's objects are a bit more 'concrete' feeling than PHP's associative arrays though).